Sources/XMTP/Push/XMTPPush.swift

```diff
@@ -0,0 +1,90 @@
+//
```

Comment:
What are the implications of having this live outside of the example folder?

It seems like if we want to treat this as a library, we are going to have to support it as such (documentation, API stability)

I wonder if it makes sense to move this into the example folder for now to lower expectations. Genuinely unsure of the right answer.

I'm not enough of a Swift expert to understand how much work that's going to put on the user to copy/paste/regenerate.

Reply:
Having it over here means it's part of the SDK and anyone can just use it.

> I wonder if it makes sense to move this into the example folder for now to lower expectations. Genuinely unsure of the right answer.

Ya this is the biggest I have as well. I really like the idea that adding push to your app is as simple as using `XMTPPush`, but it does make a hard assumption that you're running [xmtp/example-notification-server-go](https://github.com/xmtp/example-notification-server-go) somewhere.

I also like the flexibility that anyone can write any software that subscribes to the network and delivers pushes in any format they want, I just wonder if that's a lot to be asking for folks who just want to get something working, especially iOS devs who may not really know much about setting up servers.